provider/datagen/src/tests/mod.rs

```diff
@@ -9,4 +9,7 @@
 #[cfg(all(feature = "fs_exporter", feature = "use_wasm"))]
 mod make_testdata;
 
```

Comment:
Perhaps this file name is misleading; it is `provider/datagen/src/tests/mod.rs`, which is a unit test. It is not `provider/datagen/tests/mod.rs` which would be an integration test.

Reply:
aha, gotcha